packages/snfoundry/scripts-ts/deploy-contract.ts

```diff
@@ -163,17 +202,21 @@ const deployContract = async (
 
 const executeDeployCalls = async () => {
```

Comment:
missing optional maxFee as options. this is a maxFee that will be taken into account for only this function and shouldnt be overwritten on each call to `deployContract`. This one is a different parameter from the one in `declareIfNot_NotWait`